### ISSUE TITLE
Proper redirects after login

### ISSUE THREAD

**Issue by adrianbj** [processwire/processwire-requests#58]
### Short description of the enhancement

If you enter a url to a specific page in the admin and you're not logged, in would be nice if it redirected back to this page after login.

### Current vs. suggested behavior

Currently if you enter: 
```
http://mysite.com/processwire/setup/template/edit?id=29
```
you will get redirected to:
```
http://mysite.com/processwire/setup/template/?login=1&id=29
```
which is a nonsensical link

### Why would the enhancement be useful to users?

Just make it simpler to get where you're going :)

**Comment by Toutouwai** [comment]
This seems to be implemented in PW 3.0.235.

**Comment by adrianbj** [comment]
Thanks @Toutouwai - looks like it's been around for a while actually - I just tested it on 3.0.231 and it works fine.
